### DISCUSSION TITLE
3本暑假作业写不完·，已老实

### DISCUSSION THREAD

**Discussion by yusu1346** [share121/inter-knot#2510] [其他]
如何3天写完暑假作业啊

**Comment by moxvan** [comment]
写不完了，撕了把

**Comment by mapleblower** [comment]
直接说丢了，大不了挨顿揍

**Comment by wtnl551** [comment]
当然是请外援，3本作业薄的话找年纪相当或年纪大一点的人来一起写；厚的话找年纪小或动物过来，让他们把你作业糟蹋了，这样就有理由在开学时不交作业了(如果条件允许，可以搞一段作业被搞的监控录像，增加可信度）。
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hua-yuanshuai** [comment]
在绳网上已经看到许多作业写不完的了，与其在这，不如尝试拼一下，实在不行烧了吧。<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >

**Comment by Huhdfy** [comment]
这个委托我接了

**Comment by Pheathon** [comment]
只要你能花亿点小钱，我帮你肝也不是不行（）

**Comment by lllliufeng** [comment]
不要停下来啊（指写作业

**Comment by yusu1346** [comment]
写完1本了

**Comment by Robei1** [comment]
加油啊

**Comment by wwwsheepwww** [comment]
暑期末尾必备节目，还好我是毕业生

**Comment by alanyagger** [comment]
还好我大二没有暑假作业 嘻嘻

**Comment by Strix-xiao** [comment]
一支笔，一个夜晚，创造一个奇迹

**Comment by KongBai000-0** [comment]
j加油

**Comment by KongBai000-0** [comment]
加油

**Comment by shigehaoren249** [comment]
多厚的
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lixiao189** [comment]
这个委托我接了！
![image](https://github.com/user-attachments/assets/8edf37ad-4fb3-400c-b2d2-73f9714c70e5)

**Comment by Guessme404** [comment]
根本没人看的，不用写

**Comment by LBK-miq** [comment]
加油<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

不过以普遍理性而论，很多情况下寒/暑假作业的结局就是收上去以后堆放在老师办公室积灰，然后在一个合适的时机被当作废品拉走（
